### ISSUE TITLE
[FR] remote network control interface

### ISSUE THREAD

**Issue by ildar** [gateship-one/odyssey#111]
The idea and pre-discussion already in https://github.com/gateship-one/odyssey/issues/4
But here the target is lighter:
1. MPD isn't required.
2. Use a network protocol of choice to control Odyssey remotely. If that would be a subset of MP Control Protocol then even beautiful picture arise: Odyssey is enabled to control by local net and M.A.L.P. from another Android can do simple control over it. 3 basic commands would be the 80% of what's desired :)

Bottom line: MPD isn't required, just its network protocol.

**Comment by djselbeck** [comment]
I don't get it. Can you even provide one reasonable use-case? Why not just install MPD for android?

**Comment by ildar** [comment]
You're right.. For the record:
https://play.google.com/store/apps/details?id=org.musicpd
